autonomy_main_logging.py

```diff
@@ -50,25 +47,21 @@ def add_waypoint_handler(packet_contents):
     waypoint = constants.Coordinate(latitude, longitude)
     waypoints.append(waypoint)
     print("YO im addin a point")
-    outFile = open(logging, 'a')
-    tempString = time.strftime("%H%M%S") +  " Add Waypoint: lat,lon: " + str(latitude) + "," + str(longitude) + "\n"
-    outFile.write(tempString)
-    outFile.close()
+    tempString = time.strftime("%H%M%S") +  " Add Waypoint: lat,lon: " + str(latitude) + "," + str(longitude)
+    Logger.write_line(tempString)
     print("Added waypoint %s" % (waypoint,))
 
 
 def enable_autonomy(packet_contents):
     print(state_switcher.state)
     if state_switcher.state == rs.Idle():
-        with open(outString, 'a') as f:
-            f.write(time.strftime("%H%M%S") + " Enable: switching from Idle to Start\n")
+        Logger.write_line(time.strftime("%H%M%S") + " Enable: switching from Idle to Start")
         print("Yo yo de o itsd a hard-no")
         state_switcher.handle_event(rs.AutonomyEvents.START, rs.Idle())
         set_gps_waypoint()
         print("Throwing START")
     elif state_switcher.state == rs.Shutdown():
-        with open(outString, 'a') as f:
-            f.write(time.strftime("%H%M%S") + " Enable: switching from Shutdown to Restart\n")
+        Logger.write_line(time.strftime("%H%M%S") + " Enable: switching from Shutdown to Restart")
         state_switcher.handle_event(rs.AutonomyEvents.RESTART, rs.Shutdown(),  then=logging.info("Restarting Autonomy"))
```

Comment:
You have removed the logging import, is logging.info still appropriate to be used here?

Reply:
I removed the logging import from RoveComm, because I was dumb when I included it there.

Reply:
Oh, I did a from drivers.logging import LogWriter, I should probably revert it to simply import logging, but I wanted something explicit to ensure there was absolutely no chance of it being confused with one of the standard python logging utilities.

Reply:
Nevermind, I see what the issue is now, I'll run another test tomorrow and verify proper functionality, with changes as needed.

Reply:
Went back and removed the then=logging.info calls because we had sufficient comments around the state machine transitions covering the same cases. They were going to crash Autonomy.